requirements/edx/github.txt

```diff
@@ -56,9 +56,10 @@ git+https://github.com/edx/edx-lint.git@ed8c8d2a0267d4d42f43642d193e25f8bd575d9b
 -e git+https://github.com/edx/edx-reverification-block.git@1e8f5a7fd589951a90bd31a0824a2c01ac9598ce#egg=edx-reverification-block
 git+https://github.com/edx/ecommerce-api-client.git@1.1.0#egg=ecommerce-api-client==1.1.0
 -e git+https://github.com/edx/edx-user-state-client.git@30c0ad4b9f57f8d48d6943eb585ec8a9205f4469#egg=edx-user-state-client
--e git+https://github.com/edx/edx-proctoring.git@release-2015-07-29#egg=edx-proctoring
 -e git+https://github.com/edx/edx-organizations.git@release-2015-08-03#egg=edx-organizations
 
+-e git+https://github.com/edx/edx-proctoring.git@f33e09d97aff393e7de45a3d64b357ffc1d270a2#egg=edx-proctoring==0.0.4
```

Comment:
I can't recall what you had recommended last time to remediate the problem. If I remove the -e and update sandbox, would that be enough of a proof-point that this will work.

Also, can you recommend some reading on what this -e actually means? I'm having trouble understanding the theory.